app.py

```diff
@@ -214,13 +240,31 @@ def get_repository_readme(project_key: str, repo_slug: str) -> str:
     return readme_content
 
 
+def get_latest_commit(project_key: str, repo_slug: str) -> dict[str, Any]:
+    commit_path = f"projects/{project_key}/repos/{repo_slug}/commits"
+    for commit_batch in get_paginated_resource(path=commit_path, page_size=1):
+        latest_commit = commit_batch[0]
+        return latest_commit
+    return {}
+
+
 def get_repositories(project: dict[str, Any]):
     repositories_path = f"projects/{project['key']}/repos"
     for repositories_batch in get_paginated_resource(path=repositories_path):
         logger.info(
             f"received repositories batch with size {len(repositories_batch)} from project: {project['key']}"
         )
-        process_repository_entities(repository_data=repositories_batch)
+        process_repository_entities(
+            repository_data=[
+                {
+                    **repo,
+                    "latest_commit": get_latest_commit(
```

Comment:
let's use __latestCommit instead for two reasons:
1. it makes it clear that this property is not part of the repo API response
2. it uses the camelCase formatting, consistent with bitbucket's